README.md

```diff
@@ -96,25 +113,57 @@ fundamental differences in quality and design:
 To learn more about our correctness tests, please see the [Vector test harness][urls.vector_test_harness].
 
 
-## The Little Details Make All Of The Difference
+## The Little Details
 
-* **Zero delay start** - Starts and restarts without a delay.
+#### Data Model
+
+* **All data types** - [Logs][docs.data-model.log], [metrics][docs.data-model.metric], and traces (coming soon).
+* **Customizable log schema** - [Change Vector's log schema][docs.global-options#log_schema] to anything you like.
+* **Rich type system** - Support for [JSON primitive types][docs.data-model.log#types] and [timestamps][docs.data-model.log#timestamps].
+* **Metrics interoperability** - A [sophisticated metrics data model][docs.data-model.metric] ensures _correct_ interoperality between systems.
+* **Metrics aggregation** - Aggregated [histgorams][docs.data-model.metric#aggregated_histogram] and [summaries][docs.data-model.metric#aggregated_summary] allow for aggreation without loss of precision.
```

Comment:
Typo: "aggreation"

Maybe also swap out "allow for aggregation" (redundant) with something like "reduce cost/volume".